### ISSUE TITLE
Error while parsing empty lines

### ISSUE THREAD

**Issue by turizoft** [jeremyfa/yaml.js#21]
Hi,

Consider the next string: 

`"--- !ruby/hash:ActiveSupport::HashWithIndifferentAccess unit_id: - - 3 role: - disabled - regular user_id: - - 14 id: - - 42"`

Or structured:

```
--- !ruby/hash:ActiveSupport::HashWithIndifferentAccess
unit_id:
- 
- 2
role:
- disabled
- regular
user_id:
- 
- 11
id:
- 
- 32
```

While YAML.load in Ruby successfully parses it to JSON, yamljs fails somwhere.

It should generate null attributes where an empty line is found.

**Comment by jeremyfa** [comment]
Not released yet, but I have fixed this.

It will parse into:

```
{
    "unit_id": [
        null,
        2
    ],
    "role": [
        "disabled",
        "regular"
    ],
    "user_id": [
        null,
        11
    ],
    "id": [
        null,
        32
    ]
}
```

**Comment by turizoft** [comment]
Yes! It should parse into something like that. My workaround for this was to use replace on the string before parse. For may be it is useful for someone in the meanwhile, here it is:

`model.get('object_changes').replaceAll("- \n-", '- null \n-')`

In case you don't have it:

```
String.prototype.replaceAll = function(target, replacement) {
  return this.split(target).join(replacement);
};
```

**Comment by jeremyfa** [comment]
The newly released version `v0.2.0` should fix it.
